pkg/products/rhssouser/reconciler.go

```diff
@@ -301,6 +303,12 @@ func (r *Reconciler) reconcileComponents(ctx context.Context, installation *inte
 			Requests: corev1.ResourceList{corev1.ResourceCPU: k8sresource.MustParse("750m"), corev1.ResourceMemory: k8sresource.MustParse("1500Mi")},
 			Limits:   corev1.ResourceList{corev1.ResourceCPU: k8sresource.MustParse("1500m"), corev1.ResourceMemory: k8sresource.MustParse("1500Mi")},
 		}
+
+		//Set keycloak Update Strategy to Rolling as default
+		//Keycloak operator should make decision based on the image, and can change update strategy
+		//kc.Spec.Migration.MigrationStrategy = "rolling"
+		kc.Spec.Migration.MigrationStrategy = keycloak.MigrationStrategy(strings.ToLower(string(appsv1.DeploymentStrategyTypeRolling)))
```

Comment:
just a question, why do we need to to lower case `appsv1.DeploymentStrategyTypeRolling`?

I thought we wouldn't need to manipulate openshift const values

Reply:
we used low case because in Keycloak it's implemented as following:
https://github.com/keycloak/keycloak-operator/blob/357e64b67e17b8ca07072b57810d6f075e321b24/pkg/apis/keycloak/v1alpha1/keycloak_types.go
var (
	NoStrategy       MigrationStrategy
	StrategyRecreate MigrationStrategy = "recreate"
	StrategyRolling  MigrationStrategy = "rolling"
)

Reply:
I agree that we wouldn't need to manipulate Openshift const values, but it seems it will require change also from Keycloak side (where:  StrategyRolling MigrationStrategy = "rolling"), and meanwhile it can block the issue. Maybe we could suggest to  Keycloak also to use "Rolling", as in Openshift code and configuration,

Reply:
I see, I'd suggest use the `StrategyRolling` variable from keycloak operator 

```suggestion
		kc.Spec.Migration.MigrationStrategy = keycloak.StrategyRolling
```

Reply:
ok, thank you, I will try

Reply:
> I see, I'd suggest use the `StrategyRolling` variable from keycloak operator

Done